src/common.jl

```diff
@@ -18,14 +18,14 @@ end
 
 # the result type
 
-immutable Result{T}
+immutable Result{T} <: Real
```

Comment:
Correct! The result is a compound type and it is indeed just the type T that is derived from this.